service/service_test.go

```diff
@@ -309,10 +313,11 @@ func TestClose(t *testing.T) {
 
 			svcErrors := make(chan error, 1)
 			svcList := service.NewServiceList(initMock)
-			svc, err := service.Run(ctx, svcList, testBuildTime, testGitCommit, testVersion, svcErrors)
+			cfg, _ := config.Get()
```

Comment:
And again

Reply:
Again this is a test , so I guess this is fine. Please let me know otherwise.

Reply:
Please change it, if it fails because it cant access config then it is good to know it was config error and not something else (error in later assertion could hide the real issue)

Reply:
Please change it, if it fails because it cant access config then it is good to know it was config error and not something else (error in later assertion could hide the real issue)

Reply:
Please change it, if it fails because it cant access config then it is good to know it was config error and not something else (error in later assertion could hide the real issue)

Reply:
Done.